### ISSUE TITLE
players wielding axes do not disable warriors' shields

### ISSUE THREAD

**Issue by majijn** [TheCallunxz/Shadowrite#9]

**Comment by majijn** [comment]
this doesnt seem to be fixed, or is the shields not being something you can disable with axes something you deem intended?

**Comment by TheCallunxz** [comment]
Yeah, last patch was a bug fix focused patch. The things like this one and the swimming one are more features to be added later so I had closed them but I will reopen now that patch is done.
